README.md

```diff
@@ -0,0 +1,37 @@
+# TEMP_TITLE
+
+## Organizacja plików i sprawdzanie rozwiązań
+W katalogu **WDI** należy znaleźć interesujący nas zestaw oraz zadanie. Następnie w tym katalogu umieścić kod w pliku o nazwie **prog.py**
+i uruchomić plik core.py. Wynik sprawdzenia znajdzie się w pliku DATA.txt w katalogu z zadaniem.
+
+Przykładowo aby sprawdzić zadanie 7 z zestawu 2 należy w pliku **./WDI/Set_2/Task_7/prog.py** umieścić kod, a następnie w pliku po uruchomieniu
+**core.py** w pliku **./WDI/Set_2/Task_7/DATA.txt** pojawi się aktualna punktacja za zadanie.

```

Comment:
What if the order of the tasks changes? I think it's important to think of an identifier that will make it easier to find the right task even if something changes in the dataset

Reply:
The set of tasks is the set reworked at WDI, which is rather very similar every year so nothing is likely to change here. Adding an extra root directory for e.g. Python or Numpy is trivial. If we really want to be able to add new tasks, we will have to rewrite the tasks from WDI to a separate pdf and add new tasks to the end of the document

Reply:
As you said, it is very similar but doesn't have to be the same. Supposing one task is inserted in the middle of a set, every following task will have invalid tests. And that's mostly what I'm trying to avoid